pkg/vpa/vpa.go

```diff
@@ -51,6 +52,7 @@ type Reconciler struct {
 	DryRun                bool
 	IncludeNamespaces     []string
 	ExcludeNamespaces     []string
+	ignoreControllerKind  []string
```

Comment:
This field should be exported to match the other fields in this struct. This way it can be used by other packages

```suggestion
	IgnoreControllerKind  []string
```